back/engines/commercial/granular_permissions/spec/acceptance/permissions_spec.rb

```diff
@@ -30,10 +30,53 @@
       end
 
       example_request 'List all permissions of a project' do
-        expect(status).to eq(200)
-        json_response = json_parse(response_body)
+        assert_status 200
+        json_response = json_parse response_body
         expect(json_response[:data].size).to eq Permission.available_actions(@project).size
       end
+
+      example_request 'List all permissions efficiently include custom fields', document: false do
```

Comment:
Should we not document this one?

Reply:
I was thinking maybe we didn't want to repeat this in the documentation, while this spec is about things like ensuring there are no N+1 query issues. But I realize now that there would be no example of how the custom fields are included in the response, I'll add it to the docs.